src/lipalightninglib.udl

```diff
@@ -27,6 +27,56 @@ callback interface RedundantStorageCallback {
     boolean delete_object(string bucket, string key);
 };
 
+[Error]
+enum CallbackError {
+    // Invalid input.
+    // Consider fixing the input and retrying the request.
+    "InvalidInput",
+
+    // Recoverable problem (e.g. network issue, problem with en external service).
+    // Consider retrying the request.
+    "RuntimeError",
+
+    // Unrecoverable problem (e.g. internal invariant broken).
+    // Consider suggesting the user to report the issue to the developers.
+    "PermanentFailure",
```

Comment:
We already have the 3 Error types above within `LipaError`. Is there a way how we could reuse it?